bindings/pyroot/pythonizations/src/TTreePyz.cxx

```diff
@@ -166,41 +176,34 @@ PyObject *GetAttr(PyObject *self, PyObject *pyname)
 
    if (branch) {
       // found a branched object, wrap its address for the object it represents
-      auto proxy = BindBranchToProxy(tree, name, branch);
-      if (proxy != nullptr)
-         return proxy;
+      void * finalAddress = nullptr;
+      std::string finalType;
+      std::tie(finalAddress, finalType) = ResolveBranch(tree, name, branch);
```

Comment:
```suggestion
     auto [finalAddress, finalType] = ResolveBranch(tree, name, branch);
```

Reply:
I don't think that's an improvement because it hides the types. It's nice to know when reading that it's `void *` and `std::string`, no? I'll try to replace the `auto` in your suggestion with something typed, if it's not too long

Reply:
Ok, it's only working with `auto`. I'll change the variable names to make the type clearer then

Reply:
It's just a suggestion, I see your point and you can leave it as it is if you find it not necessary 👍